src/main/resources/db/changelog/changelog20221024_1559.sql

```diff
@@ -2,4 +2,7 @@
 
 -- changeset nika.avalishvili:1
 CREATE TABLE benefit (id SERIAL PRIMARY KEY,
-                        name VARCHAR(255))
+                        name VARCHAR(255),
+                        type_id INT,
+                        method_id INT)
+
```

Comment:
This changelog is supposed to be already applied on environment since these changes were already merged to main branch
So we need to apply the addition if two new columns in separate new changelog
It could be done in the same one with foreign keys

Reply:
I rearranged the changeling files and move type_id and method_id column adding operation to the changeling where the constraints are initiated.